### ISSUE TITLE
We need a better name

### ISSUE THREAD

**Issue by jackabald** [jackabald/TiDB-Hack-NL-repo-search#23]
Any ideas for a name that will suit this project?

**Comment by afzal442** [comment]
Sure, that was exactly revolving in my mind. I will think about it again. Do you have any name btw?

**Comment by jackabald** [comment]
I'm not very good at coming up with names for things, but I'm thinking like Github Lens or Code Lens or like XRayCode just because those all kind of get at the point of like you can have a huge repo and use our RAG to ask anything about it in plain English without having to go digging through docs or files kind of like an XRay.

**Comment by afzal442** [comment]
Thanks, I will also propose around those names.

**Comment by afzal442** [comment]
I would name it GitPilot or GitCodePilot or GitCodeFinderAI

**Comment by jackabald** [comment]
I like those, I think some may be a little to close to the already established GitHub Copilot, but I think they are some good ideas.

**Comment by afzal442** [comment]
so let me know which final one to pick for this project or any upgraded name if you have? Anything you can tell.

**Comment by afzal442** [comment]
so let's have this name as `gitrepo pilot`
